open-ce/build_feedstock.py

```diff
@@ -157,32 +160,31 @@ def build_feedstock(args_string=None):
     for recipe in build_config_data['recipes']:
         if args.recipes and recipe['name'] not in args.recipes:
             continue
-        for variant in utils.make_variants(args.python_versions, args.build_types, args.mpi_types):
-            conda_build_args = "conda-build "
-            conda_build_args += "--skip-existing "
-            conda_build_args += "--output-folder " + args.output_folder + " "
-            conda_build_args += "-m " + args.conda_build_config + " "
-            recipe_conda_build_config = os.path.join(os.getcwd(), "config", "conda_build_config.yaml")
-            if os.path.exists(recipe_conda_build_config):
-                conda_build_args += " -m " + recipe_conda_build_config + " "
-
-            conda_build_args += " ".join(["-c " + c + " " for c in args.channels_list])
-            conda_build_args += " ".join(["-c " + c + " " for c in build_config_data.get('channels', [])])
-
-            conda_build_args += "--variants \"" + str(variant) + "\" "
-
-            conda_build_args += recipe['path']
-
-            result = _set_local_src_dir(args.local_src_dir, recipe, recipe_config_file)
-            if result != 0:
-                break
-
-            print(conda_build_args)
-            result = os.system(conda_build_args)
-            if result != 0:
-                print("Failure building recipe: " + (recipe['name'] if 'name' in recipe else os.getcwd))
-                result = 1
-                break
+
```

Comment:
This needs to be merged with the changes added in PR #90 . There should now be a loop going over the different variants.

Reply:
yes have added it - https://github.com/open-ce/open-ce/blob/1a8f249a57288e015d56c345c63cbbc430a46d33/open-ce/build_feedstock.py#L180